### DISCUSSION TITLE
Modrinth 连通性讨论

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#4357] [讨论]
前言 #4327

**Comment by kong-que-dnf123** [comment]
省流：~”电没信用“欠大债~
测试结果就是——
移不动联不通好歹还有一大半以上能过（小闪电不关我事奥）
但事实上链接失败的就全是电没信用的.jpg（虽然也有过了的）
![image](https://github.com/user-attachments/assets/3fc72f95-6e7f-4be9-b7b5-2e479d2119da)

**Comment by Silverteal** [comment]
浙江电信可访问性好，无异常
